code/modules/cargo/packs/machinery.dm

```diff
@@ -207,6 +207,15 @@
 	crate_name = "laser mining drill crate"
 	crate_type = /obj/structure/closet/crate/engineering
 
+/datum/supply_pack/machinery/ehf_beacon
+	name = "EHF point beacon"
+	desc = "A crate containing an EHF point beacon, used to mark points of interest and semi-permanent constructions."
+	cost = 2000 // This sounds reasonable? You'll still need everything else if you want to make an outpost
```

Comment:
Niche equipment - not sure if this is too expensive or not but 1000 would feel better to me.